src/utils/json.ts

```diff
@@ -34,12 +36,24 @@ export const getJsonAsync = (path: string, defValue: any = null): Promise<any> =
     const jsonPath = join(cwd(), path)
     if (!await fileExists(jsonPath)) return resolve(defValue)
 
+    //Cache only resources
+    if (cache.has(jsonPath) && path.includes('game')) return resolve(cache.get(jsonPath))
```

Comment:
I think adding a parameter instead of checking the path would be a better choice, maybe something like
```ts
const cacheMap = new Map()

export const getJson = (path: string, defValue: any = null, useCache = false): any => {
...
if (useCache && cacheMap.has(jsonPath)) return resolve(cacheMap.get(jsonPath))
```